packaging/qpid-dispatch.spec

```diff
@@ -0,0 +1,515 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Copyright Fedora Project Authors.
+# This code is licensed under MIT license
```

Comment:
The header and following statements seem in conflict.

Things licenced to the foundation and using the ASF specific header, dont have copyright notices included, and certainly not ones attributing it to another project.

I thought your plan was to write a new file, but if this is just a lift of a Fedora file then it could simply remain MIT and be referenced as such in the LICENCE file.

Reply:
The idea is to develop the Fedora .spec in the packaged project itself, and it's continuously kept working as the project evolves. Therefore, when project releases new version, the Fedora package for it is ready to be synced into Fedora and packagers don't have to scramble to adjust to all the changes in the delta from previous packaged release.

This is ofc not relevant for dispatch, but my initial thought was that dispatch will be a proving ground and proton will be where there may be actual benefits.

Reply:
If MIT can be kept throughout, that would be best. Will do it that way.